mapr/datadog_checks/mapr/mapr.py

```diff
@@ -81,16 +81,7 @@ def check(self, _):
                 % ck_import_error
             )
 
-        try:
-            conn = self.get_connection()
-        except Exception:
-            self.service_check(
-                SERVICE_CHECK, AgentCheck.CRITICAL, self.base_tags + ['topic:{}'.format(self.topic_path)]
-            )
-            raise
-        else:
-            self.service_check(SERVICE_CHECK, AgentCheck.OK, self.base_tags + ['topic:{}'.format(self.topic_path)])
-
+        conn = self.get_connection()
```

Comment:
I don't think we should give the `get_connection` method the responsibility of reporting the health of the connection. I think we actually lose some readability by removing this section from the check method to be fair.